commons/all/src/main/java/org/infinispan/commons/marshall/ProtoStreamTypeIds.java

```diff
@@ -54,8 +54,6 @@
    int META_PARAMS_INTERNAL_METADATA = CORE_LOWER_BOUND + 3;
    int REMOTE_METADATA = CORE_LOWER_BOUND + 4;
    int UUID = CORE_LOWER_BOUND + 5;
-   int XID_IMPL = UUID + 1;
```

Comment:
@ryanemerson can we remove these? It looks like this was on added in 11.0.0.Dev4, but that was because the file was moved. I wasn't quite sure what the rules were on this yet.

Reply:
@wburns no. This was added after Dev05 build. There is not release with `XidImpl` using Protostream.

Reply:
Ah yes I see https://github.com/infinispan/infinispan/commit/70703741016dc2680adae594f99981a8dce604df#diff-2fbbcec0d882bc1f9ce78c23b927b16e now

Reply:
Compatibility only really matters between `.Final` releases as well. We don't want to restrict ourselves to much in development.

Reply:
Okay that is what I figured, just wanted to make sure.